rkt.spec

```diff
@@ -34,59 +45,68 @@ BuildRequires: libmount-devel
 BuildRequires: libxkbcommon-devel
 BuildRequires: perl-Config-Tiny
 BuildRequires: squashfs-tools
-BuildRequires: systemd
-Requires(post): systemd
-Requires(preun): systemd
-Requires(postun): systemd
+
+Requires(post): systemd => 220
+Requires(preun): systemd => 220
+Requires(postun): systemd => 220
 
 %description
 %{summary}
 
 %prep
-%setup -qn %{repo}-%{commit}
+%setup -qn %{repo}-%{version}
 
 %build
 ./autogen.sh
-%configure --with-stage1=host
+
+%configure --with-stage1=%{stage1_usr_from} --with-stage1-image-path=%{_libexecdir}/%{name}/stage1.aci
 make all
-# some issues in here prevent fedora approval
-# using COPR until then
-#GOPATH=$GOPATH:%{gopath}:$(pwd)/Godeps/_workspace:$(pwd)/_build \
-#       RKT_STAGE1_USR_FROM=usr-from-host \
-#       RKT_STAGE1_IMAGE=%{_libexecdir}/%{repo}/stage1.aci ./build
 
 %install
-# create install dirs
-install -dp %{buildroot}{%{_bindir},%{_libexecdir}/%{repo},%{_unitdir}}
-
-# install rkt binary
-install -p -m 755 build-%{name}-%{version}+git/bin/%{repo} %{buildroot}%{_bindir}
+# install binaries
+install -dp %{buildroot}{%{_bindir},%{_libexecdir}/%{name},%{_unitdir}}
 
-# install stage1.aci
-install -p -m 644 build-%{name}-%{version}+git/bin/stage1.aci %{buildroot}%{_libexecdir}/%{repo}
+install -p -m 755 build-%{name}-%{builddir}/bin/%{name} %{buildroot}%{_bindir}
+install -p -m 644 build-%{name}-%{builddir}/bin/stage1.aci %{buildroot}%{_libexecdir}/%{name}/stage1.aci
 
 # install metadata unitfiles
-install -p -m 644 dist/init/systemd/%{repo}-metadata.service %{buildroot}%{_unitdir}
-install -p -m 644 dist/init/systemd/%{repo}-metadata.socket %{buildroot}%{_unitdir}
-
-%pre
-getent group %{repo} > /dev/null || %{_sbindir}/groupadd -r %{repo}
-exit 0
+install -p -m 644 dist/init/systemd/%{name}-gc.timer %{buildroot}%{_unitdir}
+install -p -m 644 dist/init/systemd/%{name}-gc.service %{buildroot}%{_unitdir}
+install -p -m 644 dist/init/systemd/%{name}-metadata.socket %{buildroot}%{_unitdir}
+install -p -m 644 dist/init/systemd/%{name}-metadata.service %{buildroot}%{_unitdir}
+
+# Install runtime directories
+install -dp %{buildroot}%{_sharedstatedir}/%{name}
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/cas
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/cas/db
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/tmp
+install -dp %{buildroot}%{_sharedstatedir}/%{name}/containers
```

Comment:
we can add the dir perms here itself and then skip out the %attr() prepends in %files. i think %attr stuff is deprecated/not needed anymore. everything else lgtm :)